home/views/api/admin.py

```diff
@@ -576,3 +576,37 @@ def get(self, request, *args, **kwargs):
             return response
         else:
             return HttpResponse(status=401)
+
+
+class AdminUsersByAgeGroupView(View):
+    http_method_names = ["get"]
+
+    def get(self, request, *args, **kwargs):
+        if request.user.is_authenticated:
+            contest_id = request.GET.get("contest_id", None)
+            if contest_id is None:
+                return HttpResponse(status=422)
```

Comment:
On the admin homepage, it's possible to just choose arbitrary start/end dates instead of a contest. So this API endpoint should either use those dates OR the contest id.